### ISSUE TITLE
OSError Invalid cross-device link when downloading KTLint

### ISSUE THREAD

**Issue by davehadley** [macisamuele/language-formatters-pre-commit-hooks#86]
When running the KTLint pre-commit hook I see the error:
```
OSError: [Errno 18] Invalid cross-device link: '/tmp/tmpz7bctuzw' -> '/home/dave/.cache/pre-commit/ktlint0.40.0.jar'
```
I guess because on my system `/tmp` is not on the same btrfs sub-volume as `/home`.

The source of the problem is the use of `os.rename` in `language_formatters_pre_commit_hooks/utils.py`. Replacing this with `shutil.move` should fix this error.

My pre-commit yaml is:
```
  - repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
    rev: v2.1.0
    hooks:
    - id: pretty-format-kotlin
      args: [--autofix]
```

Full error message:
```
KTLint...................................................................Failed
- hook id: pretty-format-kotlin
- exit code: 1

[cwd=/home/dave/home/calorimeter] Run command: ('java', '-version')
[return_code=0] | openjdk version "11.0.9.1-internal" 2020-11-04
OpenJDK Runtime Environment (build 11.0.9.1-internal+0-adhoc..src)
OpenJDK 64-Bit Server VM (build 11.0.9.1-internal+0-adhoc..src, mixed mode)

Downloading https://github.com/pinterest/ktlint/releases/download/0.40.0/ktlint
Traceback (most recent call last):
  File "/home/dave/.cache/pre-commit/repo2681bua4/py_env-python3.9/lib/python3.9/site-packages/language_formatters_pre_commit_hooks/pretty_format_kotlin.py", line 25, in __download_kotlin_formatter_jar
    return download_url(get_url(version), "ktlint{version}.jar".format(version=version))
  File "/home/dave/.cache/pre-commit/repo2681bua4/py_env-python3.9/lib/python3.9/site-packages/language_formatters_pre_commit_hooks/utils.py", line 75, in download_url
    os.rename(tmp_file_name, final_file)
OSError: [Errno 18] Invalid cross-device link: '/tmp/tmpz7bctuzw' -> '/home/dave/.cache/pre-commit/ktlint0.40.0.jar'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/dave/.cache/pre-commit/repo2681bua4/py_env-python3.9/bin/pretty-format-kotlin", line 8, in <module>
    sys.exit(pretty_format_kotlin())
  File "/home/dave/.cache/pre-commit/repo2681bua4/py_env-python3.9/lib/python3.9/site-packages/language_formatters_pre_commit_hooks/pre_conditions.py", line 55, in wrapper
    return f(*args, **kwargs)
  File "/home/dave/.cache/pre-commit/repo2681bua4/py_env-python3.9/lib/python3.9/site-packages/language_formatters_pre_commit_hooks/pretty_format_kotlin.py", line 55, in pretty_format_kotlin
    ktlint_jar = __download_kotlin_formatter_jar(
  File "/home/dave/.cache/pre-commit/repo2681bua4/py_env-python3.9/lib/python3.9/site-packages/language_formatters_pre_commit_hooks/pretty_format_kotlin.py", line 27, in __download_kotlin_formatter_jar
    raise RuntimeError(
RuntimeError: Failed to download https://github.com/pinterest/ktlint/releases/download/0.40.0/ktlint. Probably the requested version, 0.40.0, is not valid or you have some network issue.
```

**Comment by macisamuele** [comment]
This seems a duplicate of #82 and it was fixed already in #69 but not yet released.

Could you please cross-check if the problem is gone while using the current version in master (`pip install https://github.com/macisamuele/language-formatters-pre-commit-hooks/archive/f581e56ae511b3c35b6c6856b4994f9e9f08c8ad.zip`)?

**Comment by davehadley** [comment]
Yes, `master` works. Sorry for the false report.
